data/software-wallets/metamask.ts

```diff
@@ -89,7 +90,8 @@ export const metamask: SoftwareWallet = {
       privacyPolicy: 'https://consensys.io/privacy-notice',
       transactionPrivacy: {
         defaultFungibleTokenTransferMode: 'PUBLIC',
-        stealthAddresses: notSupported,
+        [PrivateTransferTechnology.STEALTH_ADDRESSES]: notSupported,
```

Comment:
I don't understand. Are we expecting wallets to have this contract interaction baked into the wallet itself?